### ISSUE TITLE
ActiveMQ doesn't start

### ISSUE THREAD

**Issue by HansDeBal** [Alfresco/acs-deployment#266]
Hi,

When taking the docker-compose file from tag 2.1.0 and doing a docker-compose up I get the following error on the activeMq image:

`ACTIVEMQ_DATA: /opt/activemq/data
log4j:ERROR setFile(null,true) call failed.
java.io.FileNotFoundException: /opt/activemq/data/activemq.log (Permission denied)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:133)
	at org.apache.log4j.FileAppender.setFile(FileAppender.java:294)
	at org.apache.log4j.RollingFileAppender.setFile(RollingFileAppender.java:207)
	at org.apache.log4j.FileAppender.activateOptions(FileAppender.java:165)`

Could you check? Or elaborate in the documentation is something extra is needed in the config?

**Comment by alandavis** [comment]
Hi Hans,

I have just tried the same thing without getting this error. I don't think you are doing anything wrong. We have seen a few problems where the docker environment does not have sufficient memory or cpu resources. In my docker 2.1.0.5 environment I have allocated 4 cpu cores and 8 GB of memory.

If you are still having problems are you able to try a newer stable version of docker with increased resources?

Regards Alan
